eventProcessor/app.js

```diff
@@ -13,36 +15,35 @@ const createIssues = (event, hostName) => event.issues.map((issue) => ({
   ...issue,
 }));
 
-function createIssuesChunks(issues) {
-  return _.chunk(issues, 10);
-}
-
-const asyncLambdaInvoke = async (issuesChunks) => {
+const lambdaInvoke = (issuesChunks) => {
```

Comment:
Why it's not `async` anymore?

Reply:
Because the send() is returning a Promise, which the caller will use await on. So no need to do async/await here. It's duplicate.